### PR TITLE
Describe orders lifecycle

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Explains basic order life cycle - how it is created from a cart, what info is used and how it is related to payment modules.
| Fixed ticket? | #976

### CODE REVIEW

**Comment by zuk3975** [comment]
the side menu disappears when hitting new menu link created on this PR. Can anyone help with that?

File: src/content/1.7/development/orders-lifecycle/_index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Orders lifecycle
+weight: 42
+---
+
+# Order creation
+
+Orders can be created from a cart either in Front Office (FO) by the customer or in Back Office (
```
Comment (zuk3975) [inline src/content/1.7/development/orders-lifecycle/_index.md:8]: link to a order page reference when merged https://github.com/PrestaShop/docs/pull/981

**Comment by zuk3975** [comment]
@matks, Im not sure if this page is still bringing any value (most information duplicates with recently merged add new order page reference). Do you think we could modify this doc in some way so its more useful?

File: src/content/1.7/development/orders-lifecycle/_index.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/development/orders-lifecycle/_index.md:8]: https://github.com/PrestaShop/docs/pull/981 is merged 🎉

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @zuk3975

**Pull request merged**